volgen/volgen.go

```diff
@@ -40,9 +46,9 @@ func Generate(vol *volume.Volinfo) error {
 	return nil
 }
 
-// GenerateClientVolfile generates the client volfile and stores it in etcd
-func GenerateClientVolfile(vol *volume.Volinfo) error {
-	ct, err := GetTemplate(fuseTmpl, vol.GraphMap)
+// GenerateGfproxydVolfile generates the gfproxyd volfile and stores it in etcd
+func GenerateGfproxydVolfile(vol *volume.Volinfo) error {
```

Comment:
Please put the GFproxy related volgen code in a separate file. Makes it easier to maintain.

Reply:
After making the below mentioned changes, there were only 2 4-line functions remaning, that were specific to gfproxy, should i move them to different file? The best would be to have plugins for volgen as well?